.netlifyredirects

```diff
@@ -0,0 +1,6 @@
+/internal   /api
+/tomster    /mascots
+/tomster/*  /mascots/:splat
+/dasboard   /statusboard
+/guides     https://guides.emberjs.com/
```

Comment:
you don't need both trailing and non trailing slashes in the definition: https://www.netlify.com/docs/redirects/#trailing-slash

Reply:
We do need splat and non-splat though, confirm?